File: C/Exercicio37.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  
  int *num;
  int leitura, quant, i, j;

  printf("Digite a quantidade de valores a serem armazenados: ");
  scanf("%i", &quant);
  
  while(quant <=0 || quant > 20){
    printf("Erro! A quantidade de valores dever ser entre 1 e 20. Digite novamente: ");
    scanf("%i", &quant);
  }

  num = (int*)malloc(quant*sizeof(int));

  printf("Digite os valores: ");
  for(int i=0; i<quant; i++){
      scanf("%i", &num[i]);
  }

  printf("Digite um valor definido anteriormente: ");
  scanf("%i", &num[i]);

  for(int j=0; j<=leitura; j++){
    if(j == num[i]){
      printf("O numero digitado esta na posicao: ");
      printf("%i", &num[i]);
    }
  }
  return 0;
}

File: Python/Exercicio37.py
numeros = []

qtd = int(input("Digite a quantidade de valores: "))
while qtd <= 0 or qtd > 20:
    qtd = int(input("\nErro! A quantidade deve ser entre 0 e 20. Digite uma nova quantidade: "))

for i in range(0, qtd, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x)

leitura = (input("Digite um valor definido anteriormente: "))

for numero in numeros:
    if numero == leitura:
        print("O numero digitado esta na posicao:", numero)